src/unstract/sdk/vector_db.py

```diff
@@ -30,24 +30,33 @@ class VectorDB:
 
     vector_db_adapters = adapters
     DEFAULT_EMBEDDING_DIMENSION = 1536
+    EMBEDDING_INSTANCE_ERROR = (
+        "Vector DB does not have an embedding initialised."
+        "Migrate to VectorDB instead of deprecated ToolVectorDB "
+        "to use the latest features"
```

Comment:
@gaya3-zipstack Let's simply say:
```
VectorDB does not have en embedding initialised.
ToolVectorDB is deprecated. Migrate to VectorDB instead.
```

BTW, if we are throwing an error here and not allowing to proceed, then this is not actually backward compatible.
But should be relatively ok for now, let's see.

Reply:
This will not be faced by people who use the old SDK. This will be faced by people who call this function without initialising an embedding instance. ANyway, I think the error message was not very helpful. I have changed it now.